docs/api/api/required_checks.xml

```diff
@@ -0,0 +1,4 @@
+<required_checks project="home:user1">
+  <name>ci/example: unit</name>
```

Comment:
do you enforce this syntax here? Doesn't it make sense to make like

 `<check name="ci/example" unit="boot" />`

or alike?

Reply:
Nope we do not  enforce any syntax.